### PR TITLE
Create snapraid.it.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**
https://github.com/EFForg/https-everywhere/issues/15750

### CODE REVIEW

File: src/chrome/content/rules/snapraid.it.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="snapraid.it">
+	<target host="snapraid.it" />
```
Comment (J0WI) [inline src/chrome/content/rules/snapraid.it.xml]: This host does not exist, but `beta.snapraid.it` does (mismatching cert)

Comment (TotalCaesar659) [reply src/chrome/content/rules/snapraid.it.xml]: Do I have to add domains withh mismatching certs? Or just delete snapraid.it?

**Comment by pipboy96** [comment]
Please replace spaces with tabs.

**Comment by pipboy96** [comment]
Thanks!

**Comment by TotalCaesar659** [comment]
Wrong copy, please fix this example [here](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts) then.

**Comment by pipboy96** [comment]
@TotalCaesar659 This document should be completely rewritten at some point.

**Pull request merged**